src/core/VictoryDisplayApi.ttslua

```diff
@@ -0,0 +1,12 @@
+do
```

Comment:
For some reason it won't let me add the comment, but as long as you're creating the API here might as well add placeCard as well (GameKeyHandler:50).

(Note this is related since this can be considered a general victory display update/refactoring)